keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlVerificationRepository.kt

```diff
@@ -137,34 +136,38 @@ class SqlVerificationRepository(
         .fetchOneInto<VerificationState>()
     }
 
-  override fun getStates(context: VerificationContext): Map<String, VerificationState>
-    = with(context) {
-    when {
-      verifications.isEmpty() -> emptyMap() // Optimization: don't hit the db if we know there are no entries
-      else -> jooq.select(
-        VERIFICATION_STATE.VERIFICATION_ID,
-        VERIFICATION_STATE.STATUS,
-        VERIFICATION_STATE.STARTED_AT,
-        VERIFICATION_STATE.ENDED_AT
-      )
-        .from(VERIFICATION_STATE)
-        .where(VERIFICATION_STATE.ENVIRONMENT_UID.eq(environmentUid))
-        .and(VERIFICATION_STATE.ARTIFACT_UID.eq(artifact.uid))
-        .and(VERIFICATION_STATE.ARTIFACT_VERSION.eq(version))
-        .fetch()
-        .associate { (id, status, started_at, ended_at) -> Pair(id, VerificationState(status, started_at, ended_at)) }
+  override fun getStates(context: VerificationContext): Map<String, VerificationState> =
+    with(context) {
+      when {
+        verifications.isEmpty() -> emptyMap() // Optimization: don't hit the db if we know there are no entries
+        else -> jooq.select(
+          VERIFICATION_STATE.VERIFICATION_ID,
+          VERIFICATION_STATE.STATUS,
+          VERIFICATION_STATE.STARTED_AT,
+          VERIFICATION_STATE.ENDED_AT
+        )
+          .from(VERIFICATION_STATE)
+          .where(VERIFICATION_STATE.ENVIRONMENT_UID.eq(environmentUid))
+          .and(VERIFICATION_STATE.ARTIFACT_UID.eq(artifact.uid))
+          .and(VERIFICATION_STATE.ARTIFACT_VERSION.eq(version))
+          .fetch()
+          .associate { (id, status, started_at, ended_at) ->
+            id to VerificationState(status, started_at, ended_at)
```

Comment:
Metadata is not relevant when retrieving multiple state records?

Reply:
I think this is because we merged the change that added this query after I'd added metadata. I'll add it here.